tooling/ast_fuzzer/src/program/func.rs

```diff
@@ -543,6 +543,10 @@ impl<'a> FunctionContext<'a> {
             return self.gen_for(u);
         }
 
+        if freq.enabled_when("loop", self.budget > 1) {
```

Comment:
```suggestion
        if freq.enabled_when("loop", self.budget > 1 && self.unconstrained()) {
```
At least I think we can't unroll `loop` in ACIR.

Reply:
Yes, true